### PR TITLE
Job parallelism default incorrect

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1775850

### CODE REVIEW

**Comment by mburke5678** [comment]
@stencell I updated the docs to clarify the defaults for `parallelism` and `completions`. I also brought down some information from the upstream Kubernetes doc, at the suggestion of @rphillips. Are you able/willing to review these additions and changes? 

@weinliu Can you also take a look at these additions and changes?

Also, anyone have any thoughts about my breaking this topic into two: one for jobs and one for cronjobs?

Thank you both very much for any help.

**Comment by stencell** [comment]
LGTM on both enhancements and fixing the original issue in the BZ.

**Comment by weinliu** [comment]
@mburke5678 LGTM for the changes. I do not see any misleading currently for jobs and cronjobs

**Comment by mburke5678** [comment]
@openshift/team-documentation PTAL.
I brought down some upstream information around Job. Would love to make sure it sounds OK. Any thoughts on the organization of the material will be accepted. (Like, should we break this into two topics: _Jobs_ and _CronJobs_.)

File: modules/nodes-nodes-jobs-about.adoc [hunk 1]
```diff
@@ -15,6 +15,26 @@ There are two possible resource types that allow creating run-once objects in {p
 Job::
 A regular job is a run-once object that creates a task and ensures the job finishes.
 
+There are three main types of task suitable to run as a Job:
+
+* Non-parallel Jobs
```
Comment (bmcelvee) [inline modules/nodes-nodes-jobs-about.adoc]: Maybe punctuate this bullet for consistency

File: modules/nodes-nodes-jobs-about.adoc [hunk 2]
```diff
@@ -40,7 +60,7 @@ endif::[]
 ====
 A CronJob creates a job object approximately once per execution time of its
 schedule, but there are circumstances in which it fails to create a job or
-two jobs might be created.  Therefore, jobs must be idempotent and you must
+two jobs might be created.  Therefore, jobs must be idempotent and you must*
```
Comment (bmcelvee) [inline modules/nodes-nodes-jobs-about.adoc]: Single space before "Therefore"

s/must*/*must*

File: modules/nodes-nodes-jobs-about.adoc [hunk 3]
```diff
@@ -50,9 +70,12 @@ configure history limits.
 Both resource types require a job configuration that consists of the following key parts:
 
 - A pod template, which describes the pod that {product-title} creates.
-- An optional `parallelism` parameter, which specifies how many pods running in parallel at any point in time should execute a job. If not specified, this defaults to
- the value in the `completions` parameter.
-- An optional `completions` parameter, specifying how many successful pod completions are needed to finish a job. If not specified, this value defaults to one.
+- The `parallelism` parameter, which specifies how many pods running in parallel at any point in time should execute a job.
```
Comment (bmcelvee) [inline modules/nodes-nodes-jobs-about.adoc]: s/job/Job in a few places here

File: modules/nodes-nodes-jobs-about.adoc [hunk 4]
```diff
@@ -50,9 +70,12 @@ configure history limits.
 Both resource types require a job configuration that consists of the following key parts:
 
 - A pod template, which describes the pod that {product-title} creates.
-- An optional `parallelism` parameter, which specifies how many pods running in parallel at any point in time should execute a job. If not specified, this defaults to
- the value in the `completions` parameter.
-- An optional `completions` parameter, specifying how many successful pod completions are needed to finish a job. If not specified, this value defaults to one.
+- The `parallelism` parameter, which specifies how many pods running in parallel at any point in time should execute a job. 
+** For non-parallel jobs, leave unset. When unset, defaults to 1.
```
Comment (bmcelvee) [inline modules/nodes-nodes-jobs-about.adoc]: s/1/`1` in both instances

File: modules/nodes-nodes-jobs-creating.adoc [hunk 5]
```diff
@@ -34,12 +34,22 @@ spec:
...
       restartPolicy: OnFailure    <6>
 ----
-1. Optional value for how many pod replicas a job should run in parallel; defaults to `completions`.
-2. Optional value for how many successful pod completions are needed to mark a job completed; defaults to one.
-3. Optional value for the maximum duration the job can run.
-4. Option value to set the number of retries for a job. This field defaults to six.
-5. Template for the pod the controller creates.
-6. The restart policy of the pod. This does not apply to the job controller.
+1. Optionally, specify how many pod replicas a job should run in parallel; defaults to `1`.
+* For non-parallel jobs, leave unset. When unset, defaults to 1.
```
Comment (bmcelvee) [inline modules/nodes-nodes-jobs-creating.adoc]: s/1/`1` in both instances

s/job/Job - probably would help to do a find and replace for consistency

**Comment by bmcelvee** [comment]
Few tiny nits, otherwise LGTM

**Pull request merged**